spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe User, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
```

Comment:
Yo need user specs here as wells because you are generating a user model.

Reply:
Same as well. . .we only generated user model out of necessity (to make the actual cartify install work) we are not using user functionality until later feature tests.  Do you still want us to add this to our scope?  This would inherently mean we need to have multiple features tested on one PR.